### PR TITLE
#163683726 Setup Database

### PR DESCRIPTION
**Pull request by ogwurujohnson**
#### What does this PR do?
This PR sets up database for persisting data
#### Description of Task to be completed?
The following tasks where done on this PR:
- Write script for migration
- Write script for seeding
#### How should this be manually tested?
After cloning the repo, run `npm run pg`
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#163683726

### CODE REVIEW

File: server/models/db.js [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+dotenv.config();
+
+// let dbUrl;
+const nodeEnv = process.env.NODE_ENV;
+/* if (nodeEnv === 'development') {
+  dbUrl = process.env.DATABASE_URL;
+  console.log('yeah');
+} else if (nodeEnv === 'test') {
+  dbUrl = process.env.TEST_DATABASE_URL;
```
Comment (Lumexralph) [inline server/models/db.js]: @ogwurujohnson Nice work! What do you feel about having commented or dead code in your codebase?

Comment (ogwurujohnson) [reply server/models/db.js]: @Lumexralph  true, it would break code especially, if someone doesn't know why code was commented out

Comment (ogwurujohnson) [reply server/models/db.js]: lemme remove it and push back

Comment (ogwurujohnson) [reply server/models/db.js]: @Lumexralph have removed the commented codes

Comment (ogwurujohnson) [reply server/models/db.js]: please review @Lumexralph

**Review by Lumexralph — APPROVED** [review]
Good Job! This work looks good to me.

**Pull request merged**